eng/pipelines/templates/stages/archetype-java-release.yml

```diff
@@ -147,7 +147,7 @@ stages:
           - ${{if ne(artifact.options.skipPublishDocs, 'true')}}:
             - deployment: PublicDocsMs
               displayName: "Publish Updates for Docs.MS"
-              condition: and(succeeded(), ne(variables['Skip.PublishDocs'], 'true'))
+              condition: and(succeeded(), ne(variables['Skip.PublishDocs'], 'true'), ne(artifact.skipDocMs, 'true'))
```

Comment:
Why can you not do this in the if statement at template time?

Reply:
I did not skip it successfully for the first run. But it skips the step after kicking off another build.
I will move to the if clause.

Reply:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=532666&view=results

It indicates put it under if clause works well